### ISSUE TITLE
core dump on fedora rawhide

### ISSUE THREAD

**Issue by chipitsine** [PowerShell/PowerShell#7090]
Steps to reproduce
------------------

```powershell
install fedora rawhide (i.e. 29)
install powershell

```

Expected behavior
-----------------

```should work

```

Actual behavior
---------------

```
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/pscore6-docs
Type 'help' to get help.

[New Thread 0x7fff5dff9700 (LWP 7392)]
FailFast: The terminfo database is invalid.

   at System.Environment.FailFast(System.String, System.Exception)
   at System.Environment.FailFast(System.String, System.Exception)
   at Microsoft.PowerShell.UnmanagedPSEntry.Start(System.String, System.String[], Int32)
   at Microsoft.PowerShell.ManagedPSEntry.Main(System.String[])

Thread 1 "pwsh" received signal SIGABRT, Aborted.
0x00007ffff6d0100b in raise () from /lib64/libc.so.6
(gdb) bt
#0  0x00007ffff6d0100b in raise () from /lib64/libc.so.6
#1  0x00007ffff6ceb591 in abort () from /lib64/libc.so.6
#2  0x00007ffff651d1fc in PROCAbort () from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#3  0x00007ffff651c0eb in PROCEndProcess(void*, unsigned int, int) ()
   from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#4  0x00007ffff61c10d3 in SafeExitProcess(unsigned int, int, ShutdownCompleteAction) ()
   from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#5  0x00007ffff61c26f1 in EEPolicy::HandleFatalError(unsigned int, unsigned long, char16_t const*, _EXCEPTION_POINTERS*) () from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#6  0x00007ffff649ea06 in SystemNative::GenericFailFast(StringObject*, ExceptionObject*, unsigned long, unsigned int) () from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#7  0x00007ffff649f19e in SystemNative::FailFastWithException(StringObject*, ExceptionObject*) ()
   from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#8  0x00007fff7f07670b in ?? ()
#9  0x00007fff7f076700 in ?? ()
#10 0x00007fff7d168234 in ?? ()
#11 0x00007fffffffbb00 in ?? ()
#12 0x00007fff601276a8 in ?? ()
#13 0x00007fffffff9810 in ?? ()
#14 0x00007ffff627f90d in ExceptionTracker::CallCatchHandler(_CONTEXT*, bool*) ()
   from /opt/microsoft/powershell/6.1.0~preview.2/libcoreclr.so
#15 0x00007fff7d0c04c8 in ?? ()
#16 0x0000000000000000 in ?? ()
(gdb) 


```

Environment data
----------------

```powershell
> $PSVersionTable

```

**Comment by chipitsine** [comment]
pwsh crashes when TERM=xterm-256color and it works when TERM=xterm

is it possible to make pwsh working with any TERM value ?

**Comment by iSazonov** [comment]
Dup #6132

**Comment by TravisEz13** [comment]
This is resolved in 6.1.0-preview.3
